modules/game/src/main/Game.scala

```diff
@@ -110,17 +111,31 @@ case class Game(
 
   def lastMoveTimeInSeconds: Option[Int] = lastMoveTime.map(x => (x / 10).toInt)
 
-  // in tenths of seconds
-  lazy val moveTimes: Vector[Int] = BinaryFormat.moveTime read binaryMoveTimes take playedTurns
+  // in hundreds of seconds
+  lazy val clockTimes: Option[Vector[Int]] = binaryClockTimes.map(BinaryFormat.clockTimes read _)
```

Comment:
as discussed with @niklasf, so just for the record: every unit change should come with a type change, as to make the compiler detect potential bugs. `scala.concurrent.duration.FiniteDuration` can be used.